components/molecules/dropdown/components/dropdown.js

```diff
@@ -3,19 +3,32 @@
  */
 
 // React
+
+import { DropdownMenu, Icon } from '../../../'
 import React, { useEffect, useRef, useState } from 'react'
-import { array, bool, node, string, func } from 'prop-types'
+import { array, bool, func, node, oneOf, string } from 'prop-types'
 
 // Style
 import styled from 'styled-components'
 
 // UI
-import { DropdownMenu, Icon } from '../../../'
+import { Position } from 'components/theme'
```

Comment:
@mhsnprvr2 Path - same comment as earlier I am not sure it works in import

Reply:
@mhsnprvr2 Are you sure this works in PORTAL 

Please do this instead

import { DropdownMenu, Icon, Position } from '../../../'